src/install_update.ts

```diff
@@ -31,22 +32,28 @@ export interface Update {
  * updatable releases are found, dependency files are updated with the
  * latest dependency versions and the project changes are cached.
  */
-export const updateDependencies = async () => {
+export const updateDependencies = async (hookCLI: Protocol) => {
   const { releases } = await checkForSDKUpdates();
   const updatableReleases = releases.filter((r) =>
     r.update && r.current && r.latest
   );
-  const updateResp = await createUpdateResp(updatableReleases);
+  const updateResp = await createUpdateResp(updatableReleases, hookCLI);
 
   // If no errors occurred during installation, re-build
   // project as a means to cache the changes
   if (!updateResp.error) {
+    hookCLI.log(
```

Comment:
What is this extra log line being added for? Was it for testing or was the typical output unclear what was going on?

Reply:
No specific reason, indeed for testing. Should I remove it?

Reply:
I think so, as part of the log cleanup you mentioned in Slack